_includes/section-sidebar-menu.html

```diff
@@ -43,10 +43,15 @@ <h2>Menu</h2>
     {% endunless %}
   {% endfor %}
     <li>
-      <a href="/appendix/" class="{% unless locale.disable_transform %} up {%endunless%}{% if page.layout == "appendix" or page.layout == "post" %}active{% endif %}">{{ site.data['locales'][language]['sections']['appendix']}}</a>
+      <a href="/appendix/" class="{% unless locale.disable_transform %} up {%endunless%}{% if page.layout == "appendix" or page.layout == "post" %}active{% endif %}">{{ site.data['locales'][language]['sections']['appendix'] }}</a>
     </li>
+    {% unless language == 'en' %}
     <li>
-      <a href="/contributors/" class="{% unless locale.disable_transform %} up {%endunless%}{% if page.name == "contributors.html" %}active{% endif %}">{{ site.data['locales'][language]['sections']['contributors']}}</a>
+      <a href="/{{ page.lang }}/report/" class="{% unless locale.disable_transform %} up {%endunless%}{% if page.name == "report.md" %}active{% endif %}">{{ site.data['locales'][language]['sections']['translation_report'] }}</a>
```

Comment:
Hard to read. Better to break conditionald in multiple lines

Reply:
We can tackle this in another PR, the other links in the sidebar are this way as well.